src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -1424,7 +1424,7 @@ public boolean hasCitizenCompletedRequests(@NotNull final CitizenData data)
         return !getCompletedRequests(data).isEmpty();
     }
 
-    public ImmutableList<IRequest> getCompletedRequests(@NotNull final CitizenData data)
+    public ImmutableList<IRequest<?>> getCompletedRequests(@NotNull final CitizenData data)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)